sql/src/main/java/org/apache/druid/sql/calcite/expression/builtin/RoundOperatorConversion.java

```diff
@@ -32,22 +30,18 @@
 
 public class RoundOperatorConversion implements SqlOperatorConversion
```

Comment:
should we just change this to `extends DirectOperatorConversion` and drop the `toDruidExpression` part too?

Reply:
Actually good idea, I've made the changes to reflect so

Reply:
I wonder if there is any reason to keep this class?
would be using
```
new DirectOperatorConversion(SqlStdOperatorTable.ROUND, Function.Round.NAME)
```
in `DruidOperatorTable` instead of `new RoundOperatorConversion()` has any drawbacks ?

note: it seems like `UnaryFunctionOperatorConversion` doesn't have `toDruidExpressionWithPostAggOperands`  implemented - is that a problem? or we specifically don't want to allow that for `isnull` and `notnull` ? 

I now wonder why to keep the `UnaryFunctionOperatorConversion` class around as well...